openfl/plugins/processing_units_monitor/pynvml_monitor.py

```diff
@@ -3,24 +3,25 @@
 """
 pynvml CUDA Device monitor plugin module.
 
-Required package: nvidia-ml-py3
+Required package: pynvml
```

Comment:
You cat create just one additional plugin that uses `pynvml`. No need to remove the old one.

Reply:
`nvidia-ml-py3` looks like the old version of `pynvml`.
For `nvidia-ml-py3` last commit was 3 years ago.